hexrd/unitcell.py

```diff
@@ -751,8 +758,9 @@ def InitializeInterpTable(self):
                 if Z <= 92:
                     gid = fid.get('/'+elem)
                     data = np.array(gid.get('data'))
-                    self.pe_cs[elem] = interp1d(data[:, 7], data[:, 3])
-                    data = data[:, [7, 1, 2]]
+                    self.pe_cs[elem] = interp1d(data[:, WAV_ID], \
+                    data[:, MU_ID]+data[:,COH_INCOH_ID])
```

Comment:
```suggestion
                    self.pe_cs[elem] = interp1d(data[:, WAV_ID],
                        data[:, MU_ID]+data[:,COH_INCOH_ID])
```
FYI: you don't need the `\` since there are parentheses, and it's better to indent the next line to show it is continuing from the first line.